lib/eventboss/worker.rb

```diff
@@ -45,7 +45,6 @@ def terminate(wait = false)
     end
 
     def kill(wait = false)
-      stop_token
```

Comment:
Should we remove the `stop_token` method then?

Reply:
Method `#stop_token` is still used in `#terminate` and describes its job well.

Reply:
IMHO, `#terminate` method should be refactored to terminate the worker instance on which is evaluated. Unfortunately I have no idea how to do that for now.

Reply:
👍